README.md

```diff
@@ -67,7 +67,7 @@ Witnet-rust is published under the [GNU General Public License v3.0][license].
 [how to run a node]: https://medium.com/witnet/how-to-run-a-full-node-on-the-witnet-testnet-911986b8add3
 [docs]: https://docs.witnet.io
 [install]: https://docs.witnet.io/try/run-a-node/
-[Contributing]: https://docs.witnet.io/contributing
+[Contributing]: https://docs.witnet.io/community/contributing/
 [development]: https://docs.witnet.io/development
```

Comment:
@aesedepece @mariocao 
don't know whom to tag here correctly, but this link seems should be replaced with `install` one?

Reply:
Why? This is only used by the link in the "Contributions are more than welcome." sentence, right?

Reply:
I mean, I think it is perfect just in the way you updated it in your commit :wink:

Reply:
This comment is actually related to line `71` :)

Reply:
![image](https://user-images.githubusercontent.com/1465430/78050277-50fb3200-737c-11ea-9a62-9d384f0e64e5.png)

Reply:
True, I think we should separate that into three different links, one for installation, another for compilation and the last one for development.

Reply:
Ok, will split them and post update here

Reply:
I was checking how we can split those.
Installation is `try/run-a-node` obviously.
We miss compilation instructions. Should they go into documentation or be present in README.md in this repo?
What should go for development?

Reply:
Maybe "development instructions" don't sense anymore, the contributing guide is already quite extensive in that sense. What do you think?

Regarding compilation instructions, [here are some for Ubuntu][ubuntu] that I believe are up to date. Just in case it helps.

[ubuntu]: https://github.com/witnet/documentation/issues/31#issue-591847283

Reply:
Yeah, I think contributing guide, installation/running and compilation is enough.

Already spotted those compilation instruction will be adding PR to documentation with those.
Thank you